src/openrct2/paint/Supports.cpp

```diff
@@ -16,16 +16,16 @@
 
 /** rct2: 0x0097AF20, 0x0097AF21 */
 // clang-format off
-static constexpr const LocationXY8 loc_97AF20[] = {
-    {4,  4},
-    {28, 4},
-    {4,  28},
-    {28, 28},
-    {16, 16},
-    {16, 4},
-    {4,  16},
-    {28, 16},
-    {16, 28}
+static constexpr const CoordsXY SupportBoundBoxes[] = {
```

Comment:
You could have changed only the type of the `constexpr` to `CoordsXY`, it would know how to initialize itself with the initializer list above (see [example](https://godbolt.org/z/NvYi1w)). This applies to many other places in the patch.

Also if you hadn't changed the variable name, it would make it easier to review, as there would be less formatting changes due to the new width ;P

Reply:
I know, but I found using the constructors prettier. I can change it back if others agree though

Reply:
I don't think we really do it like this anywhere else in the codebase. I think it would look neater keeping the old format.

Reply:
Alright, I will change it back tonight